src/main/java/org/isf/medicalstock/gui/MovStockMultipleCharging.java

```diff
@@ -872,7 +872,9 @@ public Object getValueAt(int r, int c) {
 			} else if (c == 8) {
 				return cost;
 			} else if (c == 9) {
-				return cost.multiply(new BigDecimal(total));
+				if (cost != null)
+					return cost.multiply(new BigDecimal(total));
+				else return null;
```

Comment:
strange formatting; no braces and no need for `else` after the `return`

Reply:
Thanks!